---a/net/minecraft/item/ItemPiston.java
+++b/net/minecraft/item/ItemPiston.java
@@ -2,18 +2,16 @@
 
 import net.minecraft.block.Block;
 
-public class ItemPiston extends ItemBlock
-{
+public class ItemPiston extends ItemBlock {
+
     private static final String __OBFID = "CL_00000054";
 
-    public ItemPiston(Block block)
-    {
+    public ItemPiston(Block block) {
         super(block);
     }
 
-    public int getMetadata(int damage)
-    {
+    public int getMetadata(int damage) {
         return 7;
     }
-}
+}
